akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -48,14 +48,14 @@ private[remote] object EnvelopeBuffer {
   val TagTypeMask = 0xFF000000
   val TagValueMask = 0x0000FFFF
 
-  val VersionOffset = 0
-  val UidOffset = 4
-  val SerializerOffset = 8
-  val SenderActorRefTagOffset = 12
-  val RecipientActorRefTagOffset = 16
-  val ClassManifestTagOffset = 20
+  val VersionOffset = 0 // Int
+  val UidOffset = 4 // Long
+  val SerializerOffset = 12 // Int
+  val SenderActorRefTagOffset = 16 // Int
+  val RecipientActorRefTagOffset = 20 // Int
+  val ClassManifestTagOffset = 24 // Int
 
-  val LiteralsSectionOffset = 32
+  val LiteralsSectionOffset = 28
```

Comment:
@drewhk did you leave some extra space in front of the LiteralsSectionOffset on purpose? perhaps nice to start it on full byte?